src/OpenTelemetry.Collector.Dependencies/Implementation/System.Diagnostics.DiagnosticSource/HttpHandlerDiagnosticListener.cs

```diff
@@ -0,0 +1,987 @@
+﻿// <auto-generated />
+#if NET461
+
+/*
+ * This class is part of the System.Diagnostics.DiagnosticSource dotnet library but has unreleased changes.
+ * See: https://github.com/dotnet/runtime/pull/33732
+ * If/when that is released, this can be removed in favor of the NuGet version.
+ */
+
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
is this part of the original file? Why is it repeated three times?

Reply:
Copy paste fail, sorry! I will fix.